public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -57,6 +58,17 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
           body: new SharePanelInternally({ panelRef: vizPanel.getRef() }),
         });
 
+        scene.showModal(drawer);
+      }),
+    });
+    keybindings.addBinding({
+      key: 'p e',
+      onTrigger: withFocusedPanel(scene, async (vizPanel: VizPanel) => {
+        const drawer = new ShareDrawer({
+          title: t('share-panel.drawer.share-embed-title', 'Embed'),
+          body: new SharePanelEmbedTab({ panelRef: vizPanel.getRef() }),
```

Comment:
This won't only represent a tab anymore. Does it make sense to rename it?

Reply:
I would leave the component as it is, as we're reusing the old component. Once we enabled this new redesign 100%, we need to make these kind of changes